### PR TITLE
Done changes in the locators and added account in the excel file

### PR DESCRIPTION
**Pull request by radhikasivakumar**

### CODE REVIEW

**Comment by radhikasivakumar** [comment]
I have resolved the conflicts in TestNg Runner

File: src/test/resources/com/bizongo/leadplus/features/LP_SaveReview.feature [hunk 1]
```diff
@@ -11,15 +11,28 @@ Scenario Outline: Raising PO, Accepting PPO for Single PR and Single Dispatch Da
...
     And User Creates delivery request and Confirming timelines
     And Creating the PR   
-    Given User Opens Go Partner
-    Then Verify Go Partner "<application>"
-    Given User Selects the Purchase Requisition
-    When User Confirms all Timelines and Creates PPO
-    And User Accepts Partner Purchase Order
-    Then Verify That PPO Has Been Accepted
-    And Teardown
+    #Given User Opens Go Partner
```
Comment (sharmashashank94) [inline src/test/resources/com/bizongo/leadplus/features/LP_SaveReview.feature]: Please remove this

Comment (radhikasivakumar) [reply src/test/resources/com/bizongo/leadplus/features/LP_SaveReview.feature]: Ok sure.. Will do that..

Comment (radhikasivakumar) [reply src/test/resources/com/bizongo/leadplus/features/LP_SaveReview.feature]: removed all the commented code

**Pull request merged**